Libs/MRML/Core/vtkMRMLScalarVolumeDisplayNode.cxx

```diff
@@ -163,6 +162,10 @@ void vtkMRMLScalarVolumeDisplayNode::SetInputImageDataConnection(vtkAlgorithmOut
     vtkEventBroker::GetInstance()->AddObservation(
       inputImageDataAlgorithm, vtkCommand::ModifiedEvent, this, this->MRMLCallbackCommand );
     }
+
+  double range[2];
```

Comment:
Initialize to something (e.g., `{0.0, 0.0}` or `{0.0, 255.0}`) to ensure that you have a meaningful value even it GetDisplayScalarRange fails and returns without initializing.